src/sidebar/components/group-list-item.js

```diff
@@ -24,7 +41,8 @@ function GroupListItem({
 }) {
   const activityUrl = group.links.html;
   const hasActionMenu = activityUrl || group.canLeave;
-  const isSelectable = !group.scopes.enforced || group.isScopedToUri;
+  const isSelectable =
+    (group.scopes && !group.scopes.enforced) || group.isScopedToUri;
```

Comment:
👍 The client should always request scopes so I don't think `group.scopes` will ever actually be null. It seems sensible to check that here though given that this logic far removed from where the group fetching happens.